Clase-5-tarea2/ingresoDatos.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Ingreso Datos</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='ingresoDatos.css'>
+    
+</head>
+<body>
+    <div class="formulario">
+        <h1 class="titulo" id="titulo">Bienvenido!</h1>
+        <input type="text" class="input" id="nombre" placeholder=" Nombre"/>
+        <br>
+        <input type="text" class="input" id="segundoNombre" placeholder=" Segundo Nombre"/>
```

Comment:
Acá va `kebab-case`, y ojo con los espacios extra, hay uno dentro de cada placeholder.
```suggestion
        <input type="text" class="input" id="segundo-nombre" placeholder="Segundo Nombre"/>
```